IDE/ECLIPSE/DEOS/README.md

```diff
@@ -90,35 +83,54 @@ wolfsslPort
 
 </processTemplate>
 ```
-Depending on your configuration, wolfSSL uses upto four mutexes. You also need to configure enough memory for the stack of each threads and the process logical memory pool.
-
-
- 2. Right click on the `udp-vs-tcp` project, select properties and add the following macros in the DDC-I Options > C Compile > Preprocessor
-      -   DEOS_ALLOW_OBSOLETE_DEFINITIONS
-      -   WOLFSSL_USER_SETTINGS
- 3.  Add the following directory paths in the DDC-I Options > C Compile > Directories and in the DDC-I Options > C++ Compile > Directories
-      -   $(PROJECT_DIR)/wolfsslPort
-      -   $(PROJECT_DIR)/wolfsslPort/wolfssl
-      -   $(PROJECT_DIR)/wolfsslPort/IDE/ECLIPSE/DEOS
-      -   $(PROJECT_DIR.printx)/code
- 4.  Change the optimization level in the DDC-I Options > C Compile > Code Generation > Optimization level:g
-      -   g
- 5.  Add the following library dependencies in the DDC-I Options > Deos > Dependencies
-      -   math
-      -   dart
-      -   ansi
-      -   printx
-          - You must add printx into your workspace, File >DDC-I Deos example project > training > printx
- 6.  Edit $(PROJECT_DIR)/wolfsslPort/IDE/ECLIPSE/DEOS/user_setting.h to customize your configuration. For example, you can undef or define these tests.
-      -   #undef NO_CRYPT_TEST
-      -   #undef NO_CRYPT_BENCHMARK
-      -   #undef NO_WOLFSSL_CLIENT
-      -   #undef NO_WOLFSSL_SERVER
- 7.  Edit your application source file where main() thread is defined and add the following:
-      -   #include "printx.h"
-      -   #include "tls_wolfssl.h"
-      -   and a call to `wolfsslRunTests()`
+
+Depending on your configuration, wolfSSL uses upto four mutexes. You
+also need to configure enough memory for the stack of each threads and
+the process logical memory pool.
+
+5. Right click on the `udp-vs-tcp` project, select properties and add
+   the following macros in the DDC-I Options > C Compile >
+   Preprocessor
+
+  * WOLFSSL_USER_SETTINGS
+
+6.  Add the following directory paths in the DDC-I Options > C Compile >
+    Directories and in the DDC-I Options > C++ Compile > Directories
+
+  * $(PROJECT_DIR.wolfssl)/../../../..
```

Comment:
Double-check the three paths. Having wolfssl appended seems odd.  "$(PROJECT_DIR.wolfssl)"

Reply:
Confirmed this is correct, this allows referring to other project directories.